nio.conf

```diff
@@ -267,6 +264,28 @@ port=[[PKS_PORT]]
 #
 #websocket_ping_interval=10
 
+# auth
+
+# Module that provides authentication
+#auth_module          = pubkeeper.server.core.auth.local.LocalAuthModule
+
+# Class that will provide token validation
+#auth_provider = pubkeeper.server.core.auth.local.LocalAuthProvider
+
+# Below are example settings using LocalAuth.  Modules and Providers
+# will be provided this directive for their own configuration
+
+# validate is the URL used by LocalAuthProvider to lookup the provided token
+# for local environments set PKS_AUTH_VALIDATE_URL=http://localhost:8002/validate
+#auth_validate = [[PKS_AUTH_VALIDATE_URL]]
+
+# IP Address of the Auth Server
+#auth_ip_address      = 0.0.0.0
```

Comment:
This is no longer needed.

Reply:
Yeah, I'm reorganizing this, stand by for a push. It will affect the pk module code too